src/main/java/com/pirimid/cryptotrade/DTO/PlaceOrderResDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.pirimid.cryptotrade.DTO;
+
+
+import lombok.*;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Builder
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class PlaceOrderResDTO {
+    String id;
+    Double price;
+    Double size;
+    String symbol; // BTC/ETH
+    String side;
+    String type;
+    Date createdAt;
+    Double executed_amount;
```

Comment:
Can you use Capitalization in variable name?

Reply:
Okay We will do that.

Reply:
Did you mean executedAmount or for every variable ?